stages/org.osbuild.dmverity

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import sys
+import tempfile
+import osbuild.api
+
+
+def open_dmverity(blocksize, partlabel, datapart, hashpart, mountpoint):
+    hash = subprocess.run(["/usr/sbin/veritysetup",
+                            "format", datapart, hashpart,
+                            "--data-block-size", f"{blocksize}"],
+                           check=True, capture_output=True)
+    for line in hash.stdout.decode().splitlines():
+      if line.startswith("Root hash:"):
+        hash = line.split(":", 1)[1].strip()
+        break
+
+    subprocess.run(["/usr/sbin/veritysetup",
```

Comment:
Why this setup/mount here? AFAICT it's enough to run `veritysetup format` ? If the setup/mount is needed would you mind adding a comment in the code about the "why" ?

Reply:
`open` + `mount` are here to ensure we can use this partition in the future without any issues.